portfolio/src/main/webapp/script.js

```diff
@@ -29,11 +29,20 @@ function addRandomGreeting() {
 
 function addMessage() {
   fetch('/data').then(response => response.json()).then((message) => {
```

Comment:
it would make sense to rename `message` into `messages` to reflect there can be multiple messages